steward/mew.toml

```diff
@@ -0,0 +1 @@
+keystore = "mew"
```

Comment:
Seems like maybe this was checked in by accident?

Reply:
I was using those for tests. Removing them now